shinyswatch/picker/theme_picker.js

```diff
@@ -0,0 +1,81 @@
+// Get the source path of the shinyswatch-js script
+// to figure out where the htmldependency has ended up
+// e.g. src="lib/shinyswatch-js-5.3.1/bootstrap.bundle.min.js"
+// avoids having to know the shinswatch bootstrap version or lib location
+const sw_script = document.querySelector('script[src*="shinyswatch-js"')
+const subdir = sw_script.src
+  .replace(/\/bootstrap.*js$/, '')
+  .replace('shinyswatch-js', 'shinyswatch-all-css')
+
+const display_warning = setTimeout(function () {
+  window.document.querySelector('#shinyswatch_picker_warning').style.display =
+    'block'
+}, 1000)
+
+Shiny.addCustomMessageHandler('shinyswatch-hide-warning', function (message) {
+  window.clearTimeout(display_warning)
+})
+
+Shiny.addCustomMessageHandler('shinyswatch-refresh', function (message) {
+  window.location.reload()
+})
+
+Shiny.addCustomMessageHandler('shinyswatch-pick-theme', function (theme) {
+  const base_dir = `${subdir}/${theme}`
+
+  const sheets = ['bootswatch.min.css', 'shinyswatch-ionRangeSlider.css']
```

Comment:
Let's add some commentary here about where in the python logic this should mirror